tests/unit/test_charm.py

```diff
@@ -10,11 +10,20 @@
 
 from charm import SmtpIntegratorOperatorCharm
 
+MINIMAL_CHARM_CONFIG = {
+    "host": "smtp.example",
+    "port": 25,
+}
+MINIMAL_CHARM_CONFIG_WITH_PASSWORD = {
+    **MINIMAL_CHARM_CONFIG,
+    "password": "somepassword",  # nosec
```

Comment:
I think we can use `secrets.token_hex()` here :D

Reply:
yes please - that way we don't get into the habit of disabling security warnings in our code. As a side note, we do have a standard to explain why a disable is reasonable if it is required

Reply:
Fixed